common/src/state/call.rs

```diff
@@ -37,9 +37,8 @@ pub struct Call {
     pub id: Uuid,
     pub conversation_id: Uuid,
     pub participants: Vec<DID>,
-    pub participants_joined: Vec<DID>,
+    pub participants_joined: HashMap<DID, ParticipantState>,
     pub participants_speaking: HashMap<DID, Instant>,
-    pub participants_state: HashMap<DID, ParticipantState>,
```

Comment:
You did a great job with PR description (unlike me :laughing:) so I hate to ask this but it would be good to summarize the specific changes like this one so people new to the code (like me) can more easily understand what's going on. It will also help future archeologist (which may be you :smile:).

Reply:
@sdwoodbury Thanks! Now please also help the future archeologists. :)